app/assets/main/react_components/footprint_form/ResultPage.jsx

```diff
@@ -23,6 +23,10 @@ const ResultPage = ({ result, texts, lang, page, onPageChange, currency }) => {
     const stripePromise = loadStripe('pk_test_4QHSdRjQiwkzokPPCiK33eOq')
     const commonStrings = texts.commonText
 
+    if (texts.registrationsText.accept_policies===undefined) {
```

Comment:
```suggestion
    if (texts.registrationsText.accept_policies === undefined) {
```

Tycker detta är snyggare, men det är en smaksak!